.github/workflows/stale.yml

```diff
@@ -0,0 +1,36 @@
+name: Stale Questions
+
+on:
+  schedule:
+  - cron: "0 0 * * *"
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/stale@v2
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        days-before-stale: 10
+        days-before-close: 4
```

Comment:
I think those numbers are far too short. If we are moving from having no automation to having those date limits, we should start being A LOT more flexible with the dates, and gradually decrease the time limits as we fine-tune the process.

Reply:
I think that's after adding the label `stat: need more info`. Do you think it should be more than 14 days @engelgabriel ?

Reply:
I don't think two weeks is too aggressive to the close an issue AFTER an interaction.. and again, it will receive a new comment from the bot after 10 days in case the owner misses the notification from first interaction.. it will be closed after 4 days after the **second** interaction..

I'm not saying those are the right numbers. I'm just explaining exactly what will happen.. also I found other repos using the same params and found them reasonable. do you have something else in mind?

Reply:
IF.. I said... IF an user opened an issue and we answered/asked in time...7 days is enough time to interact again... I mean, the user should be more interested than us to solve his problem and probably he is waiting for some reply...

Reply:
I think it should be at least longer than the Brazilian mandatory vacations days duration.